tests/trace/test_scores.py

```diff
@@ -0,0 +1,42 @@
+from concurrent.futures import Future
+
+import weave
+from weave.trace.feedback_types.score import SCORE_TYPE_NAME
+from weave.trace.weave_client import get_ref
+from weave.trace_server import trace_server_interface as tsi
+
+
+def test_send_score_call(client):
+    @weave.op
+    def my_op(x: int) -> int:
+        return x + 1
+
+    @weave.op
+    def my_score(input_x: int, model_output: int) -> int:
+        return {"in_range": input_x < model_output}
+
+    call_res, call = my_op.call(1)
+    assert call_res == 2
+    score_res, score_call = my_score.call(1, call_res)
+    assert score_res == {"in_range": True}
+    res_fut = client._send_score_call(call, score_call)
+    assert isinstance(res_fut, Future)
+    res = res_fut.result()
+    assert isinstance(res, str)
```

Comment:
May be worth commenting -- this wasn't obvious on first read.

It seems like what's happening is:
1. Invoke an op
2. Invoke a scoring op
3. "attach" a score_call to the call?

Reply:
ok